lib/apiservers/engine/backends/vicbackends.go

```diff
@@ -45,9 +47,12 @@ var (
 	productVersion      string
 
 	vchConfig *config.VirtualContainerHostConfigSpec
+
+	insecureRegistries []string
+	RegistryService    *registry.Service
 )
 
-func Init(portLayerAddr, product string, config *config.VirtualContainerHostConfigSpec) error {
+func Init(portLayerAddr, product string, config *config.VirtualContainerHostConfigSpec, registries []url.URL) error {
```

Comment:
These registries are used as insecure registries - call the arg that to make it explicit